packages/11ty/_includes/components/figure/video/print.js

```diff
@@ -15,7 +15,7 @@ module.exports = function(eleventyConfig) {
 
   const { figureLabelLocation, imageDir } = eleventyConfig.globalData.config.params
 
-  return function({ aspect_ratio: aspectRatio, caption, credit, id, label, mediaType, poster}) {
+  return function({ aspect_ratio: aspectRatio, caption, credit, id, label, mediaType, poster=''}) {
```

Comment:
I was testing out static video figures to ensure the media type exclusion for lightbox slides was working and noticed this would throw errors if a static video figure did NOT have a poster

Reply:
A warning when a video does not have a `poster` image defined would be helpful.